client/src/pages/Details.js

```diff
@@ -13,13 +13,18 @@ export default function Details({ member, updateMember, members }) {
     remember: '',
   };
 
+  //Takes current state of the member out of the members Array.
+  const [newMember, setNewMember] = useState(
+    members.find((newMember) => newMember.id === member.id)
```

Comment:
Maybe you can encapsulate this into its own function with a descriptive name and thus won't need the comment in line 16
`const findCurrentMember = (newMember) => members.find(member => member.Id === newMember.id)`